### ISSUE TITLE
warnings in combination with typescript

### ISSUE THREAD

**Issue by PSeitz** [asapach/babel-plugin-rewire-exports#23]
I get a lot of errors of the kind below. In my setup I'm also using @babel/plugin-transform-typescript.

```
The exported identifier "_ObjectKind" is not declared in Babel's scope tracker
as a JavaScript value binding, and "@babel/plugin-transform-typescript"
never encountered it as a TypeScript type declaration.
It will be treated as a JavaScript value.

This problem is likely caused by another plugin injecting
"_ObjectKind" without registering it in the scope tracker. If you are the author
 of that plugin, please use "scope.registerDeclaration(declarationPath)".
```

**Comment by sergei-startsev** [comment]
Hi @PSeitz. Can you provide a minimal program that reproduces this warning?

**Comment by CodyRay** [comment]
I have observed the same errors in another typescript project, [here is a simple repro.](https://github.com/haroldhues/babel-plugin-rewire-exports-23-reproduction) I found that the error only occurred when using default exports or `const` exports with `unsafeConst: false`.

I found this [similar issue](https://github.com/babel/babel/pull/10302), but I am not sure how to apply a similar fix to this plugin

**Comment by asapach** [comment]
Thanks for the repro, @haroldhues!
I've published v2.1.0 - please try it out.
